modules/knative-service-cluster-local.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm-custom-domains.adoc
+
+[id="knative-service-cluster-local_{context}"]
+= Setting cluster availability to `cluster-local`
+
+By default, Knative services are published to a public IP address.
+Being published to a public IP address means that Knative services are public applications, and have a publicly accessible URL.
+
+Publicly accessible URLs are accessible from outside of the cluster.
+However, developers may need to build back-end services that are only be accessible from inside the cluster, known as _private services_.
+// Cluster administrators can configure private services for the cluster so that all services are private by default.
+// Need to add additional details about editing the configmap for admins
+Developers can label individual services in the cluster with the `serving.knative.dev/visibility=cluster-local` label to make them private.
+
+.Procedure
+
+* Set the visibility for your service by adding the `serving.knative.dev/visibility=cluster-local` label:
++
+
+[source,terminal]
+----
+$ oc label ksvc <service_name> serving.knative.dev/visibility=cluster-local
+----
+
+.Verification steps
+
+* Check that the URL for your service is now in the format `http://<service_name>.<namespace>.svc.cluster.local`, by entering the following command and reviewing the output:
```

Comment:
You can put a `\` at the front of the example URL to cancel out creating the link in the rendered version:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#example-urls